onnxruntime/core/providers/cpu/quantization/quantize_linear.cc

```diff
@@ -86,6 +120,59 @@ REGISTER_DEQUANTIZELINEAR_VERSIONED(int8_t)
 REGISTER_DEQUANTIZELINEAR_VERSIONED(uint8_t)
 REGISTER_DEQUANTIZELINEAR_VERSIONED(int32_t)
 
+#if !defined(DISABLE_CONTRIB_OPS)
+namespace contrib {
+
+// Register alternate MS domain versions of the DequantizeLinear kernel.
+// The MS domain versions additionally support 16-bit integer quantization types.
+ONNX_CPU_OPERATOR_TYPED_MS_KERNEL(
+    DequantizeLinear,
+    1,
+    uint8_t,
+    KernelDefBuilder()
+        .TypeConstraint("T1", DataTypeImpl::GetTensorType<uint8_t>())
+        .TypeConstraint("T2", DataTypeImpl::GetTensorType<float>()),
+    DequantizeLinear<uint8_t>);
```

Comment:
Why do we need these for types supported by ONNX? When would a model use the contrib op domain in that case?

Reply:
These 8-bit contrib op versions were added in 2019 by https://github.com/microsoft/onnxruntime/pull/1712
> When these 2 ops were promoted to onnx domain they were removed from contrib ops because they were fairly new and ORT did not have support for quantization earlier so there were no models out there using this op. However these 2 ops in particular were used by winml's weight packing script and therefore winml had produced models using the contrib op version. So need to add these back.

This PR moved them into this file.